src/custom/components/WalletModal/WalletModalMod.tsx

```diff
@@ -219,6 +221,15 @@ export default function WalletModal({
         //   const walletAddress = await connector.getAccount()
         //   logMonitoringEvent({ walletAddress })
         // })
+        .then(() => {
+          // fix for this https://github.com/gnosis/cowswap/issues/1930
+          // manually set the WalletConnectConnector http.connection.url to currently connected network url
```

Comment:
ubermeganit: I would prefer to see the comments in the inverse order. 

// Manually set the ...
// Fixes http:....

Reply:
nice! detectiv-hack work!